content.js

```diff
@@ -143,6 +144,37 @@ function removeChatFromLocalStorage(href) {
 }
 
 function removeSavedChat(href, savedChatsList) {
+    const div = document.createElement('div');
+    div.className = 'alert-overlay';
+    div.innerHTML = `
+      <div class="dialog-wrapper scale p-2 dark:bg-gray-800 bg-gray-50">
+        <div class="flex items-center pb-2 border-b border-gray-200 dark:border-gray-700 gap-2 mb-2">
+          <span class="del-btn">${deleteIcon}</span>
+          <p class="text-gray-200 font-semibold text-lg">Are you sure?</p>
+        </div>
+        <span class="text-sm leading-[25px] text-gray-600 dark:text-gray-400">Do you really want to delete this saved chats? This process can't be undone.</span>
```

Comment:
Try to include the name of the chat that is being deleted instead of only saying do you want to delete this saved chat
![image](https://github.com/user-attachments/assets/a5a44b50-cd2d-4cc4-acdf-a17cea65c862)